sway-lsp/src/handlers/request.rs

```diff
@@ -19,11 +13,17 @@ use std::{
 use sway_types::{Ident, Spanned};
 use sway_utils::PerformanceData;
 use tower_lsp::jsonrpc::Result;
+use tower_lsp::lsp_types::{
+    CodeLens, CompletionResponse, DocumentFormattingParams, DocumentSymbolResponse,
+    InitializeResult, InlayHint, InlayHintParams, PrepareRenameResponse, RenameParams,
+    SemanticTokensParams, SemanticTokensRangeParams, SemanticTokensRangeResult,
+    SemanticTokensResult, TextDocumentIdentifier, Url, WorkspaceEdit,
+};
 use tracing::metadata::LevelFilter;
 
 pub fn handle_initialize(
     state: &ServerState,
-    params: &lsp_types::InitializeParams,
+    params: &tower_lsp::lsp_types::InitializeParams,
```

Comment:
these are getting pretty long now. It's probably nicer so either include the type in the use statement directly or `use tower_lsp::lsp_types::{self, .... }` to keep these how they were.